trycourier/client.py

```diff
@@ -26,7 +27,13 @@ def __init__(self,
           timeout (float|tuple): Timeout in seconds. (Connect, Read) Defaults
           to 5 seconds for both.
         """
-        self.base_url = base_url
+        if not base_url:
+            if 'COURIER_BASE_URL' in environ:
```

Comment:
I think this can be simplified. If there's not a base_url passed in, grab it from the environment variable with the default url as the second param instead of none.